apps/sms/views/inbox/test/unit/inbox_test.js

```diff
@@ -93,9 +96,9 @@ suite('thread_list_ui', function() {
         '<li id="thread-' + i + '" class="threadlist-item" ' +
             'data-time="' + someDate + '" ' +
             'data-thread-id="' + i + '">' +
-          '<label>' +
-            '<input type="checkbox" data-mode="threads" value="' + i + '"/>' +
-          '</label>' +
+          '<gaia-checkbox data-mode="threads" value="' + i + '">' +
```

Comment:
gaia-header is a different case: it does not inherit any behavior from anything other than a normal DOM Element.

But I realize gaia-checkbox is encapsulating the label as well.

TBH I don't realize the value of such components yet. But well.